### PR TITLE
 Ensembling Experiment Support Added

### PR DESCRIPTION
**Pull request by ArthurBoschet**
## Ensembling Experiment Support Added

This PR enhances this repository by adding the possibility to evaluate ensemble of different models.

### Changes

- **Script Addition:** A new script, `nnunet_scripts/ensemble_inference_and_evaluation.sh`, has been added. This script facilitates generating probabilities from each model, which are then collectively ensembled to create a prediction which is evaluated.
  
- **Script Modification:** The `nnunet_scripts/run_evaluation.py` script has been modified to eliminate the need for the `--mapping_path` argument when it is not necessary.

### CODE REVIEW

File: nnunet_scripts/ensemble_inference_and_evaluation.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# The first argument should be the path to the datasets.
```
Comment (hermancollin) [inline nnunet_scripts/ensemble_inference_and_evaluation.sh:1]:
```suggestion
# The first argument should be the path to the folder containing the folds to ensemble
```

IIUC this is what you meant? Or maybe I misunderstood the README

**Review by hermancollin — CHANGES_REQUESTED** [review]
Typo in docstring of _ensemble_inference_and_evaluation.sh_ script but otherwise everything LGTM

File: nnunet_scripts/ensemble_inference_and_evaluation.sh (hunk 1 above)
Comment (ArthurBoschet) [reply nnunet_scripts/ensemble_inference_and_evaluation.sh:1]: @hermancollin Actually neither are correct. it should be the nnunet folder path that is provided as the first argument to the script.

Comment (ArthurBoschet) [reply nnunet_scripts/ensemble_inference_and_evaluation.sh]: I fixed it now!

**Review by hermancollin — APPROVED** [review]

**Pull request merged**